docs/conf.py

```diff
@@ -10,7 +10,7 @@
 
 
 docs_dir = os.path.abspath(os.path.dirname(__file__))
-base_dir = os.path.join(docs_dir, "..")
+base_dir = os.path.abspath(os.path.join(docs_dir, ".."))
```

Comment:
On a related note, in a Python 3.4+ project one could just do:

```python
from pathlib import Path
base_dir = Path(__file__).parent.resolve()
```

When are we dropping Python 2 :stuck_out_tongue_closed_eyes: ?